xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -238,17 +242,20 @@ extern "C"
 
     uint32_t m_flags;
 
-    char m_name[256]; /*!< @brief (optinal) name of the stream, \0 for default handling */
-    char m_codecName[32]; /*!< @brief (required) name of codec according to ffmpeg */
+    char m_name
+        [INPUTSTREAM_MAX_STRING_NAME_SIZE]; /*!< @brief (optinal) name of the stream, \0 for default handling */
+    char m_codecName
+        [INPUTSTREAM_MAX_STRING_CODEC_SIZE]; /*!< @brief (required) name of codec according to ffmpeg */
```

Comment:
You could avoid this awkward spacing by moving the doxy to before the symbol

```c++
//! @brief (optional) name of the stream, \0 for default handling
char m_name[INPUTSTREAM_MAX_STRING_NAME_SIZE];

//! @brief (required) name of codec according to ffmpeg
char m_codecName[INPUTSTREAM_MAX_STRING_CODEC_SIZE];
```

Or however the doxy is best formatted. Also: optinal -> optional

Reply:
Have changed it, yes much better then as clang not do his line breaks inside.